rewrite-yaml/src/test/java/org/openrewrite/yaml/AppendToSequenceTest.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.yaml;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.yaml.Assertions.yaml;
+
+class AppendToSequenceTest implements RewriteTest {
+
+    @Test
+    void appendToSequenceHasDashTrue() {
+        rewriteRun(
+          spec -> spec
+                .cycles(1)
```

Comment:
Why is it necessary for you to customize the number of cycles the test harness runs for?
If this is done correctly it should be stable, making all changes the first time it visits a file and no further changes on any subsequent visit.

Reply:
Yes, you are correct.  I fixed an issue in the AppendToSequenceVisitor last night.  This allowed me to delete the cycles(1) customizations.